.github/workflows/deploy.yml

```diff
@@ -1,25 +1,26 @@
 name: Deploy
-
 concurrency:
-  group: ${{ github.event.repository.name }}-deploy
+  group: deploy
   cancel-in-progress: true
-
 on:
   workflow_dispatch:
   push:
     branches:
       - main
-
 jobs:
-  Docker_Hub:
+  ci:
+    name: CI
+    uses: ./.github/workflows/ci.yml
```

Comment:
Actually there does appear to be one functional change here: looks like `report`'s `deploy.yml` was not calling `ci.yml` like all the others do, now it does. When I get around to it I'm gonna consider removing the `ci.yml` call from `deploy.yml` if it seems safe to do so, but I'll make that change in the cookiecutter and apply it to all projects.